go.mod

```diff
@@ -4,8 +4,8 @@ go 1.18
 
 require (
 	github.com/IOTechSystems/onvif v0.1.5
-	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.5
-	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.5
+	github.com/edgexfoundry/device-sdk-go/v3 v3.0.0-dev.6
+	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.6
```

Comment:
Leave as `v3.0.0-dev.5`. Other changes in new version we are not ready for yet and the SDK is using `v3.0.0-dev.5`